pages/the_hague.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Website about The Hague</title>
+
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css"
+        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" href="../css/hague.css">
+    <link rel="icon" href="../images/favicon-32x32.webp" type="image/x-icon">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Anton&display=swap" rel="stylesheet">
+    <link
+        href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,200;0,300;0,400;0,500;0,700;1,400&display=swap"
+        rel="stylesheet">
+</head>
+<body>
+    <main>
+        <header class="header">
+            <div class="header_logo">
+              <a href="../index.html">
+                <img src="../images/logo_en.svg" alt="Logo the Netherlands"></a>
+            </div>
+            <ul class="header_navigation">
+              <li><a href="./pages/amsterdam.html">Amsterdam</a></li>
+              <li><a href="./pages/rotterdam.html">Rotterdam</a></li>
+              <li><a href="the_hague.html">The Hague</a></li>
+              <li><a href="./pages/utrecht.html">Utrecht</a></li>
```

Comment:
Links don't work at the header. I slightly changed the the url, with this change it should be working.
Also I suggest deleting link to The Haag page, because we are already here =)
```suggestion
              <li><a href="./amsterdam.html">Amsterdam</a></li>
              <li><a href="./rotterdam.html">Rotterdam</a></li>
              <li><a href="./utrecht.html">Utrecht</a></li>
```
If I am not mistaken @E-CFG-Dev you need to confirm that agree to these changes on your side, in case you want to include them in this merge

Reply:
Hello Irina, I solve the problem of The Hague page, changed from  
<li><a href="the_hague.html">The Hague</a></li> to
<li><a href= "./pages/the_hague.html">The Hague</a></li>